src/pages/index.tsx

```diff
@@ -3,34 +3,45 @@ import React from 'react';
 import Layout from '../components/layout/layout';
 import SEO from '../components/seo';
 import { Grid, GridItem } from '../components/grid/grid';
+import { PageTitle, SectionTitle } from '../components/typography/typography';
+import styled from 'styled-components';
+import { up } from '../components/breakpoint/breakpoint';
+
+const Title = styled(PageTitle)`
```

Comment:
Confusing as you mix general styles (PageTitle) with local components that sound similar general (Title). But good for a first round 💪